frontend/src/features/permits/pages/TermOversize/components/pay/PermitPayFeeSummary.tsx

```diff
@@ -14,6 +16,14 @@ export const PermitPayFeeSummary = ({
   permitType?: PermitType;
   onPay: () => void;
 }) => {
+  const [inputAction, setInputAction] = useState(false);
+  const debouncedInputAction = useDebounce(inputAction, DEBOUNCE_TIMEOUT);
+
```

Comment:
I'm not sure if we should create a reusable hook that encapsulates this useDebounce(), and pass in any arbitrary action as input. Something to consider in case we need to debounce other buttons in the near future...

Reply:
It's ok, we can create a reusable hook if and when we need one in the future.